ui/src/main/resources/com/cloudbees/workflow/controller.jelly

```diff
@@ -11,6 +11,7 @@
     </st:documentation>
     <div class="cbwf-stage-view">
         <div cbwf-controller="${name}" objectUrl="${rootURL}/${it.target.url}" fragCaption="${fragCaption}" />
-        <st:adjunct includes="org.jenkinsci.pipeline.stageview_adjunct"/>
```

Comment:
Are https://github.com/jenkinsci/pipeline-stage-view-plugin/blob/master/ui/src/main/js/stageview_adjunct.js and https://github.com/jenkinsci/pipeline-stage-view-plugin/blob/master/ui/src/main/less/stageview_adjunct.less unused now?

Reply:
Thanks for pointing it out, I guess so, lemme remove those files.